<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>33</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="closeButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../qapitrace.qrc">
       <normaloff>:/resources/dialog-close.png</normaloff>:/resources/dialog-close.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="prevButton">
     <property name="toolTip">
      <string>Find previous occurance of the string</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../qapitrace.qrc">
       <normaloff>:/resources/go-up-search.png</normaloff>:/resources/go-up-search.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextButton">
     <property name="toolTip">
      <string>Find next occurance of the text</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../qapitrace.qrc">
       <normaloff>:/resources/go-down-search.png</normaloff>:/resources/go-down-search.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="caseSensitiveBox">
     <property name="text">
      <string>Case Sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="regexBox">
     <property name="text">
      <string>&amp;Regex</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="notFoundLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;b&gt;Text not found&lt;/b&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qapitrace.qrc"/>
 </resources>
 <connections/>
</ui>
